### ISSUE TITLE
Compare to runs from exam IN5550

### ISSUE THREAD

**Issue by pmhalvor** [pmhalvor/fgsa#16]
Saga not working at all any more.

- [x] Run IN5550 on fox
- [x] get close to same results for best model 
> - Pipeline using pretrained Transformer
> - first epoch gave best score (although close to 10 epochs)
> - very high learning rate compared to what I've seen
- [x] quick hyperparameter search 
- [x] transfer findings to FgsaLSTM and IMN
